osm2gtfs/tests/README.md

```diff
@@ -8,8 +8,8 @@ Accra
 There are 3 unittests for the Accra GTFS generation
 
 1. Generation of the cache stops Data from an Overpass result Mock
-* Generation of the cache routes Data from an Overpass result Mock
-* Generation of the GTFS ZIP File using the previously generated cache data
+1. Generation of the cache routes Data from an Overpass result Mock
```

Comment:
I kindly suggest to rephrase a bit for better language and usage of upper case words. My proposal (feel free to take, adapt or use anything else):

> * Obtain mock-up data on stops from OpenStreetMap and cache it.
> * Obtain mock-up data on routes from OpenStreetMap and cache it.
> * Generate GTFS file from using the previously generated cache data. (remove the "and" from the last list item).